core/domain/exp_domain_test.py

```diff
@@ -428,7 +429,7 @@ def test_validation(self):
             'Terminal interactions must not have a default outcome.')
 
         interaction.id = 'TextInput'
-        interaction.default_outcome = None
```

Comment:
Just a thought for the future -- should we be making some of these properties private, so that they cannot be modified directly? If so, may be worth filing an issue for that.